DrewBradley/rancid-tomatillos
# FEATURE // Potatoes

### What does this PR do?
Adds a potato-based rating system to the individual view.

### What does it fix?
This PR fixes the lack of potatoes in the individual view.

### Is this a feature or a fix?
Both. It fixed the problem of missing potatoes and features more potatoes.

### Where should the reviewer start?
In individual view, confirm that potatoes are rending properly.
Lines 25-27 of ```IndividualView.js``` and corresponding CSS.

### How should this be tested?
When the application is launched and you view an individual movie, you should see those sweet Yukon Golds display.
